docs/features/deploying/manage-projects.md

```diff
@@ -2,6 +2,40 @@
 title: Manage Projects
 ---
 
+## Hostname / Alias
+
+The **Alias** feature gives the ability to add the same KintoBlock more than once in a single project by changing the hostname. The changes made to the original KintoBlock will be applied to all the aliases.
```

Comment:
Host names can only be set prior to deploying the block. If you wish to change a blocks name, you must undeploy it first then redeploy, or deploy the same block with a different hostname first, then remove the old one.